templates/cli/lib/questions.js.twig

```diff
@@ -368,9 +369,9 @@ const questionsPushCollections = [
     },
     {
         type: "input",
-        name: "override",
-        message: 'Are you sure you want to override this collection? This can lead to loss of data! Type "YES" to confirm.'
-    },
+        name: "changes",
+        message: `Changes above will cause recreation of an attribute. All existing documents will lose data in those attributes. Type "YES" to confirm`
```

Comment:
Previously the message was about overriding the collection. Now the message has been changes to something specific like attribute creation. Is this correct? 

What if there is a deletion of attribute, or changes to indexes? This `message` does not talk about those scenarios